db/migrate/20200514132247_add_stripe_id_uniq_to_stripe_invoices.rb

```diff
@@ -0,0 +1,8 @@
+class AddStripeIdUniqToStripeInvoices < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
+
+  def change
+    remove_index(:stripe_invoices, :stripe_id)
+    add_index(:stripe_invoices, :stripe_id, unique: true)
```

Comment:
Shouldn't there be added a validation to the model as well, so we can gracefully fail with a validation error instead of crashing with an index error?